src/function/pragma/pragma_queries.cpp

```diff
@@ -58,10 +58,27 @@ string PragmaFunctionsQuery(ClientContext &context, const FunctionParameters &pa
 
 string PragmaShow(ClientContext &context, const FunctionParameters &parameters) {
 	// PRAGMA table_info but with some aliases
-	return StringUtil::Format(
-	    "SELECT name AS \"column_name\", type as \"column_type\", CASE WHEN \"notnull\" THEN 'NO' ELSE 'YES' "
-	    "END AS \"null\", NULL AS \"key\", dflt_value AS \"default\", NULL AS \"extra\" FROM pragma_table_info('%s');",
-	    parameters.values[0].ToString());
+
+	// Remove leading and trailing quotes from string
+	auto table_name = parameters.values[0].ToString();
+	StringUtil::RemoveFirstAndLastIfMatching(table_name, '"');
+
+	return StringUtil::Format("SELECT name AS \"column_name\", "
```

Comment:
Can we use a multi-line string literal here, similar to what is used in the [FTS extension](https://github.com/duckdb/duckdb/blob/master/extension/fts/fts_indexing.cpp#L37)?